test/Feature/MaximalReconvergence/subgroup_uniform_control_flow.test

```diff
@@ -1,22 +1,31 @@
 #--- source.hlsl
-RWStructuredBuffer<uint> Out : register(u0);
+RWStructuredBuffer<float> Out : register(u0);
 
 [numthreads(8,1,1)]
 void main(uint3 TID : SV_GroupThreadID) {
+    float maxActiveLaneCount = min(WaveGetLaneCount(), 8);
```

Comment:
I would leave this with `WaveActiveSum(1)` so we have a working test. No need for a comment.

This is not testing the wave intrinsics, so as long as we have something that is suppose to work we are fine.